internal/weldr/api.go

```diff
@@ -2370,6 +2365,18 @@ func (api *API) composeInfoHandler(writer http.ResponseWriter, request *http.Req
 		reply.Uploads = targetsToUploadResponses(compose.ImageBuild.Targets, composeStatus.State)
 	}
 
+	// Add package dependencies from the compose
+	// This information may not be included for the compose
+	if _, ok := compose.PackageSets["packages"]; ok {
+		dependencies := compose.PackageSets["packages"]
```

Comment:
Could we just store the `packages` PackageSet in the store? That way we don't make the names of the sets API, and we would also avoid storing the build-root packages etc which we currently have no use for / way to export.